Shared/View/CreateActivityView.swift

```diff
@@ -49,6 +52,17 @@ struct CreateActivityView: View {
                 }
                 TextField("Name", text: $newFriend) {
                     self.friendList.append(self.newFriend)
+                    Task {
+                        do {
+                            newFriendID = try await UserViewModel.user(named: self.newFriend)?.id ?? ""
```

Comment:
1. I don't believe this fixes the issue, but please let me know if it does
2. Do we need to store newFriendID as a state?